docs/developer/getting-started/create-edit-forms.md

```diff
@@ -0,0 +1,101 @@
+# Create/Edit Forms
+
+# Standard Create/Edit Form Architecture
```

Comment:
For a getting started guide there's a lot of extra, in-depth information here that's not all entirely relevant to basic create/edit component

Reply:
To be fair, there's a lot of code we are supposed to reuse that goes beyond how create/edit forms are normally implemented in Vue. It would help if you could specify what should be removed.